exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/export_as_is.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>"has <strong>a</strong> utility function ..." if it's just one function, or "has some utility <strong>functions</strong> ..." if there are few of them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>fixed in all notebooks</p>